templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+	<h1>Delete Manufacturer</h1>
+	<p>Are you sure you want to delete manufacturer {{ manufacturer }}</p>
+	<p><i>All Cars with this manufacturer will be also deleted!</i></p>
+	<form action="" method="post">
```

Comment:
The `action` attribute in the form tag is empty, which means the form will submit to the current URL. Ensure this behavior is intended and that the view handling this submission is correctly set up to process the form data.